From a276fc8fea92c3233faf9bd9ca579ced2ef58071 Mon Sep 17 00:00:00 2001
From: lolmaxlik <ilya.lebedev.2003@mail.ru>
Date: Tue, 16 Aug 2016 12:31:03 +0600
Subject: [PATCH] patch freq

---
 product/qcom-perf.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/product/qcom-perf.mk b/product/qcom-perf.mk
index ddccd6b..e0b8035 100644
--- a/product/qcom-perf.mk
+++ b/product/qcom-perf.mk
@@ -14,7 +14,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 
 ifeq ($(TARGET_BOARD_PLATFORM_VARIANT),msm8916)
 PRODUCT_PROPERTY_OVERRIDES += \
-    ro.min_freq_0=800000
+    ro.min_freq_0=400000
 else
 ifeq ($(TARGET_BOARD_PLATFORM_VARIANT),msm8939)
 PRODUCT_PROPERTY_OVERRIDES += \
-- 
2.7.4

